### ISSUE TITLE
Ooyala - Get video from sbnation.com

### ISSUE THREAD

**Issue by MaryCoding** [rg3/youtube-dl#4859]
Hi, I am wanting to get a video from [sbnation.com](http://www.sbnation.com/) but currently `youtube-dl` is not able to. I noticed there main video player is supported by [Ooyala](http://www.ooyala.com/). Therefore, it should work since `youtube-dl` has support for [Ooyala](http://www.ooyala.com/). I tried this two ways:

Using the original [sbnation.com](http://www.sbnation.com/) link: 
```
youtube-dl -v -o "/media/%(title)s.mp4"   http://www.sbnation.com/lookit/2015/2/3/7969559/michael-jordan-punched-will-perdue-chicago-bulls
[debug] System config: []
[debug] User config: []
[debug] Command-line args: ['-v', '-o', '/media/%(title)s.mp4', 'http://www.sbnation.com/lookit/2015/2/3/7969559/michael-jordan-punched-will-perdue-chicago-bulls']
[debug] Encodings: locale UTF-8, fs UTF-8, out UTF-8, pref UTF-8
[debug] youtube-dl version 2015.02.03.1
[debug] Python version 2.7.6 - Linux-3.13.0-35-generic-x86_64-with-Ubuntu-14.04-trusty
[debug] exe versions: ffmpeg git-2014-12-27-d4fd3f2, ffprobe git-2014-12-27-d4fd3f2
[debug] Proxy map: {}
[generic] michael-jordan-punched-will-perdue-chicago-bulls: Requesting header
WARNING: Falling back on generic information extractor.
[generic] michael-jordan-punched-will-perdue-chicago-bulls: Downloading webpage
[generic] michael-jordan-punched-will-perdue-chicago-bulls: Extracting information
ERROR: Unsupported URL: http://www.sbnation.com/lookit/2015/2/3/7969559/michael-jordan-punched-will-perdue-chicago-bulls
Traceback (most recent call last):
  File "/usr/local/bin/youtube-dl/youtube_dl/extractor/generic.py", line 694, in _real_extract
    doc = parse_xml(webpage)
  File "/usr/local/bin/youtube-dl/youtube_dl/utils.py", line 1501, in parse_xml
    tree = xml.etree.ElementTree.XML(s.encode('utf-8'), **kwargs)
  File "/usr/lib/python2.7/xml/etree/ElementTree.py", line 1300, in XML
    parser.feed(text)
  File "/usr/lib/python2.7/xml/etree/ElementTree.py", line 1642, in feed
    self._raiseerror(v)
  File "/usr/lib/python2.7/xml/etree/ElementTree.py", line 1506, in _raiseerror
    raise err
ParseError: not well-formed (invalid token): line 85, column 111
Traceback (most recent call last):
  File "/usr/local/bin/youtube-dl/youtube_dl/YoutubeDL.py", line 619, in extract_info
    ie_result = ie.extract(url)
  File "/usr/local/bin/youtube-dl/youtube_dl/extractor/common.py", line 268, in extract
    return self._real_extract(url)
  File "/usr/local/bin/youtube-dl/youtube_dl/extractor/generic.py", line 1110, in _real_extract
    raise UnsupportedError(url)
UnsupportedError: Unsupported URL: http://www.sbnation.com/lookit/2015/2/3/7969559/michael-jordan-punched-will-perdue-chicago-bulls
```

Second, I tried using the  [Ooyala](http://www.ooyala.com/) player link:
```
youtube-dl -v -o "/media/%(title)s.mp4"   http://player.ooyala.com/player.swf?player=e7e908eebffd4efeb96a3096aa9b4bd0
[debug] System config: []
[debug] User config: []
[debug] Command-line args: ['-v', '-o', '/media/%(title)s.mp4', 'http://player.ooyala.com/player.swf?player=e7e908eebffd4efeb96a3096aa9b4bd0']
[debug] Encodings: locale UTF-8, fs UTF-8, out UTF-8, pref UTF-8
[debug] youtube-dl version 2015.02.03.1
[debug] Python version 2.7.6 - Linux-3.13.0-35-generic-x86_64-with-Ubuntu-14.04-trusty
[debug] exe versions: ffmpeg git-2014-12-27-d4fd3f2, ffprobe git-2014-12-27-d4fd3f2
[debug] Proxy map: {}
[generic] player: Requesting header
[redirect] Following redirect to http://player.ooyala.com/static/cacheable/947272d312bbb31971ab9ee0b28b5a1f/player_v2.swf?player=e7e908eebffd4efeb96a3096aa9b4bd0
[generic] player_v2: Requesting header
WARNING: Falling back on generic information extractor.
[generic] player_v2: Downloading webpage
WARNING: URL could be a direct video link, returning it as such.
[debug] Invoking downloader on 'http://player.ooyala.com/static/cacheable/947272d312bbb31971ab9ee0b28b5a1f/player_v2.swf?player=e7e908eebffd4efeb96a3096aa9b4bd0'
[download] Destination: /media/sf_linux_sandbox/player_v2.mp4
[download] 100% of 499.29KiB in 00:00
```

**Comment by naglis** [comment]
Support for Ooyala embeds on SBN has been added and will be available in the next release of youtube-dl. Thanks for the report.

Until the new version is released, you may fetch the video in question by running:
```bash
youtube-dl 'http://player.ooyala.com/player.js?embedCode=55Zjl3cjpqk6KD3w3nfhK9amvTORUhf9' 
```

**Comment by phihag** [comment]
I believe this issue to be fixed in youtube-dl 2015.02.09 and newer. See [our FAQ](https://yt-dl.org/update) if you need help updating.
